x-element.js

```diff
@@ -1918,18 +1872,51 @@ class TemplateEngine {
     }
   }
 
-  // Attach a document fragment into some container. Note that all the DOM in
-  //  the fragment will already have values correctly bound.
-  static #inject(result, node, options) {
+  // Inject a given result into a node for the first time. If we’ve never seen
+  //  the template “strings” before, we also have to generate html, parse it,
+  //  and find out binding targets. Then, we commit the values by iterating over
+  //  our targets. Finally, we actually attach our new DOM into our node.
+  static #inject(result, node, before) {
```

Comment:
@klebba — Did some refactoring / renaming here since you called it out as confusing when you were looking things over yesterday. Hope it helps!